Assets/Scripts/Core/Controllers/SaveController.cs

```diff
@@ -17,7 +17,7 @@ public class SaveController : MonoBehaviour {
 			if(Input.GetKeyDown(KeyCode.Backslash)) {
 				StartCoroutine(getSaveNameForSaving());
 			}
-			if(Input.GetKeyDown(KeyCode.Pipe)) {
+			if(Input.GetKeyDown(KeyCode.Equals)) {
```

Comment:
Did you update this keycode change in the mapeditor help text? If not, please do so.